### PR TITLE
feature #2696 - chat message after wallet send

### LINKED ISSUE #2696: No send message shown in chat if sending from wallet
### Description

*Type*: Bug

*Summary*: send Funds to a contact from wallet -> no new send message is shown in the chat as well as no chat created if this is the very fisrt message

#### Expected behavior
new send message is shown in the chat

#### Actual behavior
no new messages are shown in the chat

### Reproduction
- Open Status
- Open Wallet
- Tap Send
- Type some valid amount, e.g. 1234
- From Contacts select Jarrad
- tap on Sign transaction and proceed

### Additional Information
* Status version: last develop
* Operating System: Android, iOS

### PR DESCRIPTION
**Pull request by goranjovic**
fixes #2696

### Summary:

Sends a message in 1:1 chat every time user send a transaction (ETH or token) to a contact, whether the transaction was sent from chat or wallet. 

### Review notes (optional):

This PR removes `:execute-stored-command` since that was  temporary workaround implemented so that messages could be sent after the command was interrupted by a shortcut. Since we now have unified message handling for both chat and wallet transactions alike, that kludge is no longer needed.

### Testing notes (optional):

Transactions sent by pasting the address (or scanning the QR) will still not appear in chat history and will not display recipient name in transaction history (only address). This will be handled in a separate PR for issue #2884 

This PR removes the root cause of all of those "message with data from old transaction" bugs.

### Steps to test:
- Send transactions from chat (message should appear)
- Send transactions from wallet to contacts (message should appear)
- Send transactions from wallet to non-contacts or by address/qr (message should not appear)
- Check any of the conditions that triggered bugs where a message was sent with old transaction data or before it was updated.

status: ready

### CODE REVIEW

**Review by janherich — APPROVED** [review]

File: src/status_im/chat/models/message.cljs [hunk 1]
```diff
@@ -475,3 +476,36 @@
...
+                             :color         "#5fc48d",
+                             :ref           ["transactor" :command 83 "send"],
+                             :name          "send",
+                             :params        [{:name "asset", :type "text", :placeholder "Currency"}
+                                             {:name "amount", :type "number", :placeholder "Amount"}],
+                             :type          :command,
+                             :async-handler false,
+                             :icon          "money_white",
+                             :scope         #{:global :personal-chats :registered :humans},
+                             :title         "Send transaction",
```
Comment (flexsurfer) [inline src/status_im/chat/models/message.cljs]: i18n

File: src/status_im/chat/models/message.cljs [hunk 2]
```diff
@@ -475,3 +476,36 @@
...
       (send-command cofx params))))
+
+(defn custom-send-command-message [whisper-id address asset amount]
+  {:message  nil,
+   :command  {:command      {:description   "Send a payment",
+                             :bot           "transactor",
+                             :color         "#5fc48d",
+                             :ref           ["transactor" :command 83 "send"],
+                             :name          "send",
+                             :params        [{:name "asset", :type "text", :placeholder "Currency"}
```
Comment (flexsurfer) [inline src/status_im/chat/models/message.cljs]: i18n for all params placeholders

File: src/status_im/chat/models/message.cljs [hunk 3]
```diff
@@ -475,3 +476,36 @@
 
       :else
       (send-command cofx params))))
+
+(defn custom-send-command-message [whisper-id address asset amount]
+  {:message  nil,
+   :command  {:command      {:description   "Send a payment",
```
Comment (flexsurfer) [inline src/status_im/chat/models/message.cljs]: i18n

File: src/status_im/ui/screens/navigation.cljs [hunk 4]
```diff
@@ -70,23 +70,26 @@
...
 
+(def navigation-interceptors
+  [unload-data-interceptor (re-frame/enrich preload-data!)])
+
 ;; event handlers
 
 (handlers/register-handler-db
  :navigate-to
- [unload-data-interceptor (re-frame/enrich preload-data!)]
+ navigation-interceptors
```
Comment (flexsurfer) [inline src/status_im/ui/screens/navigation.cljs:80]: 👍

**Review by flexsurfer — APPROVED** [review]

File: src/status_im/chat/models/message.cljs (hunk 2 above)
Comment (goranjovic) [reply src/status_im/chat/models/message.cljs]: Actually, let me see if this can be removed. It shouldn't be required for the message

**Review by dmitryn — APPROVED** [review]

**Comment by asemiankevich** [comment]
1. new chat is not created and no `/send` message sent when sending eth/token from wallet to address I scanned (iOS -> Android, Android -> iOS). 
2. new chat is not created and no `/send` message sent when sending eth/token from wallet to address I copypasted (iOS -> Android, Android -> iOS). 
In both scenarios, there is no difference I have added devices to contacts or not. This is expected to an outlined limitation in the PR notes.

what I can confirm working here:

1. Sending from wallet to someone in my contacts - new chat is created, transaction message appears
2. Requesting from wallet works as expected 
3. Sending from old to new - no message send. Sending from new to old - no message as well
4. Release upgraded to this PR - send message appears

I think it is good to merge

**Pull request merged**